<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Sudoku</class>
 <widget class="QMainWindow" name="Sudoku">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sudoku</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="Board" name="board" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuGame">
    <property name="title">
     <string>Игра</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionClean"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuMode">
    <property name="title">
     <string>Сложность</string>
    </property>
    <addaction name="actionEasy"/>
    <addaction name="actionMedium"/>
    <addaction name="actionHard"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>Инфо</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="plugins">
    <property name="title">
     <string>Плагины</string>
    </property>
   </widget>
   <widget class="QMenu" name="update">
    <property name="title">
     <string>Обновления</string>
    </property>
   </widget>
   <addaction name="menuGame"/>
   <addaction name="menuMode"/>
   <addaction name="menuAbout"/>
   <addaction name="plugins"/>
   <addaction name="update"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionEasy">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Легко</string>
   </property>
   <property name="toolTip">
    <string>Лёгкая сложность</string>
   </property>
  </action>
  <action name="actionMedium">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Средне</string>
   </property>
   <property name="toolTip">
    <string>Средняя сложность</string>
   </property>
  </action>
  <action name="actionHard">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Сложно</string>
   </property>
   <property name="toolTip">
    <string>Сложно</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>Новая игра</string>
   </property>
   <property name="toolTip">
    <string>начать новую игру</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Выход</string>
   </property>
   <property name="toolTip">
    <string>Выход из игры</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>О программе</string>
   </property>
   <property name="toolTip">
    <string>О программе</string>
   </property>
  </action>
  <action name="actionClean">
   <property name="text">
    <string>Очистить</string>
   </property>
   <property name="toolTip">
    <string>Очистить доску</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>цвет фона</string>
   </property>
  </action>
  <action name="actionplugins">
   <property name="text">
    <string>plugins</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Board</class>
   <extends>QWidget</extends>
   <header>Board.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
